gdal/swig/python/scripts/gdal_retile.py

```diff
@@ -699,6 +700,7 @@ def Usage():
     print('        [-s_srs srs_def]  [-pyramidOnly] -levels numberoflevels')
     print('        [-r {near/bilinear/cubic/cubicspline/lanczos}]')
     print('        [-useDirForEachRow]')
+    print('        [-overwrite overwrite existing tiles in targetDir]')
```

Comment:
that's not the write way of declaring the option. Should be just [-overwrite] here.
But apps/gdal_utilities.dox should be updated to add the new option and document it